scanner/indexer/testdata/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Can we just add `/testdata` in `../.gitignore` instead? Also, maybe better to create/delete `/tmp` directories.